README.rst

```diff
@@ -396,6 +396,33 @@ Usage example as a context-manager:
             signal=openwisp_signal,
         )
 
+``openwisp_utils.tests.TimeLoggingTestRunner``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``TimeLoggingTestRunner`` logs speed of all tests that are run and reports the slow
+tests which took more than the threshold time (0.3s-default) in the end.
```

Comment:
Also, a picture is worth 1000 words. Can you add a screenshot of the [travis build](https://travis-ci.com/github/nepython/openwisp-monitoring/jobs/348881143#L1037) you have linked? It communicates pretty well at first sight what it does